src/TraceEvent/Ctf/CtfTracing.Tests/CtfTracing.Tests.csproj

```diff
@@ -36,19 +36,8 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System" />
+    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework" />
   </ItemGroup>
-  <Choose>
```

Comment:
> Also I need this to work in 2015 too.

Removing support for 2015 was an intentional design change. It allows the code to be cleaned up and use new language features moving forward. I believe this is not problematic for the following reasons:

1. Visual Studio 2017 is freely available for contributors
2. Visual Studio 2015 and 2017 have very different approaches to .NET Core projects which are not compatible
3. Visual Studio 2017 introduces C# 7, some features of which have proven useful for improving application performance and overall code quality in various situations